pkg/operator/ceph/cluster/osd/status.go

```diff
@@ -235,6 +235,21 @@ func (c *Cluster) completeOSDsForAllNodes(config *provisionConfig, configOSDs bo
 				currentTimeoutMinutes++
 				if currentTimeoutMinutes == timeoutMinutes {
 					config.addError("timed out waiting for %d nodes: %+v", remainingNodes.Count(), remainingNodes)
+					//start to remove remainingNodes waiting timeout.
+					for remainingNode := range remainingNodes.Iter() {
+						if err := c.updateOSDStatus(
+							remainingNode,
+							OrchestrationStatus{Status: OrchestrationStatusCompleted},
+						); err != nil {
+							config.addError("failed to set orchestration completed status for timeout node %s: %+v", remainingNode, err)
+							continue
+						}
+						if err := c.kv.ClearStore(
```

Comment:
A couple small points:
- The common pattern (at least in Rook) is that the `if` statement is on one line if possible
- If you're calling another function, do it on a separate line for better readability.

So it might look like this:
```
nodeName := k8sutil.TruncateNodeName(orchestrationStatusMapName, remainingNode)
if err := c.kv.ClearStore(nodeName); err != nil {
   config.addError("removing timeout node %s error: %+v", remainingNode, err)
}
```

Reply:
good to heard that there are patterns to follow, thanks